src/styles/signup-header.styles.css

```diff
@@ -0,0 +1,30 @@
+.signup-header {
+    width: 100%;
+    height: 65px;
+    margin-top: 35px;
+    flex-direction: row;
+    position: absolute;
+    font-family: 'Montserrat';
```

Comment:
is this being imported correctly? i dont see any change to index.js to incorporate this font. it's showing on my screen, and your screenshot as default font not montserrat.